File: score.c
#include <stdio.h> 

int main(){ 

    char name[20]; 
    char title[][20] = {"국어", "수학", "영어"}; 
    int score[3] = {0}; 
    int sum = 0; 
    int i; 
    double avg; 

    printf("학생 이름 입력: "); 
    scanf("%s", name); 

    for(i = 0; i < 3; i++){ 
        printf("점수 입력: "); 
        scanf("%d", &score[i]); 
        sum += score[i]; 
    }

    avg = sum / 3.0; 
    printf("%s 학생의 합계는 %d이고\n", name, sum); 
    printf("평균은 %.2f 입니다.", avg); 

    return 0; 

}

File: Java/J091-92/src/Score.java
public class Score {
    private int kor,eng,mat;
    private int sum;
    private double avg;


    public Score(int kor, int eng, int mat) {
        this.kor = kor;
        this.eng = eng;
        this.mat = mat;
        sum = kor + eng + mat;
        avg = (double)(kor+eng+mat) / 3.0;
    }

    public int getKor() {
        return kor;
    }

    public void setKor(int kor) {
        this.kor = kor;
    }

    public int getEng() {
        return eng;
    }

    public void setEng(int eng) {
        this.eng = eng;
    }

    public int getMat() {
        return mat;
    }

    public void setMat(int mat) {
        this.mat = mat;
    }

    public int getSum() {
        return sum;
    }

    public void setSum(int sum) {
        this.sum = sum;
    }

    public double getAvg() {
        return avg;
    }

    public void setAvg(double avg) {
        this.avg = avg;
    }

    public String toString(){
        String s = "";
        s += " - Total sum: " + getSum() + " Total average: " + getAvg();
        return s;
    }
}
